airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt

```diff
@@ -86,6 +86,15 @@ create table "public"."connection_operation"(
       "operation_id"
     )
 );
+create table "public"."feature_flag"(
+  "id" uuid not null,
+  "name" varchar(256) not null,
+  "value" varchar(2147483647) null,
```

Comment:
Why's this set so high instead using the default infinite for varchar or limiting it to 256 or similar?

Reply:
It looks like max Int to me. This file is an autogenerated one which got created when running the dumpSchema command, I defined it without a limit here: https://github.com/airbytehq/airbyte/pull/10014/files#diff-3a3d6d3d328fb108a672f34a61ecfb6335a08c9f9f5953e1bf95c1911aadf0d2R37

Reply:
Oh forgot this was autogenerated. nvm.